game/scripts/vscripts/Sections/Chapter3/Chapter3.ts

```diff
@@ -598,7 +598,7 @@ const onStart = (complete: () => void) => {
             playerCanMoveNeutralFromBackpack = true
             goalSwapItems.start()
             highlightUiElement(firstNeutralSlotUIPath)
-            highlightUiElement(secondNeutralItemUIPath)
+            highlightUiElement(secondNeutralItemUIPath, undefined, HighlightMouseButton.Right)
```

Comment:
This might be confusing: it should be left, but it is also to drag.

Reply:
Should add double-click and drag later